llm/binding_smoke_test.go

```diff
@@ -20,6 +21,12 @@ func TestDeepcodeLLMBinding_Explain_Smoke(t *testing.T) {
 		WithLogger(&logger),
 	)
 	outputChain := make(chan string)
-	err := binding.Explain(context.Background(), AIRequest{Id: uuid.New().String(), Input: "{}"}, HTML, outputChain)
+	endpoint, errEndpoint := url.Parse(defaultEndpointURL)
+	if errEndpoint != nil {
+		// time to panic, as our default should never be invalid
+		panic(errEndpoint)
+	}
```

Comment:
Suggest using the assert package for this

```suggestion
	assert.NoError(t, err)
```